src/server/test/web/readingsBarGroupFlow.js

```diff
@@ -9,23 +9,112 @@
 
 const { chai, mocha, app } = require('../common');
 const Unit = require('../../models/Unit');
-// const { prepareTest,
-//     parseExpectedCsv,
-//     createTimeString,
-//     expectReadingToEqualExpected,
-//     getUnitId,
-//     ETERNITY,
-//     METER_ID,
-//     unitDatakWh,
-//     conversionDatakWh,
-//     meterDatakWh } = require('../../util/readingsUtils');
+const { prepareTest,
+    parseExpectedCsv,
+    createTimeString,
+    expectReadingToEqualExpected,
+    getUnitId,
+    ETERNITY,
+    METER_ID,
+    GROUP_ID,
+    unitDatakWh,
+    conversionDatakWh,
+    meterDatakWh } = require('../../util/readingsUtils');
 
 mocha.describe('readings API', () => {
     mocha.describe('readings test, test if data returned by API is as expected', () => {
         mocha.describe('for bar charts', () => {
             mocha.describe('for quantity groups', () => {
```

Comment:
This is not your doing but could you change quantity -> flow since this is what this file does.